### ISSUE TITLE
sorting the names

### ISSUE THREAD

**Issue by Darrkzero** [Oluwasetemi/altschool-opensource-names#558]
the names should be sorted alphabetically when the refresh button is clicked

**Comment by Oluwasetemi** [comment]
Good suggestion. What are your ideas around achieving this?

**Comment by Darrkzero** [comment]
using node.js to read and edit the text.txt file should be able to achieve it

**Comment by Oluwasetemi** [comment]
You can check the sorted.js file. I have an implementation for that and it includes read and writing with nodejs. Let me know what should be done. Next. Thanks for jumping on this issue.

**Comment by Oluwasetemi** [comment]
I will close this issue @Darrkzero. Waiting for your response.

**Comment by Darrkzero** [comment]
the sorted.txt file is perfect.  You can create a different button to switch from text.txt file to sorted.txt file to make the names appear in alphabetical order.

**Comment by Oluwasetemi** [comment]
Yes that will be a great one but we could use JavaScript to sort it on the fly and replace instantly.

**Comment by Darrkzero** [comment]
That will also work.

**Comment by manlike-samm** [comment]
Attempted this in #743. Kindly review and give feedback. Also, i don't know if the ordered.js and sorted.txt files are needed anymore.
